script/main.js

```diff
@@ -0,0 +1,110 @@
+// toggle menu
+const toggleMenu = () => {
+    let menu = document.querySelectorAll('.menu-mobile');
+    let active = menu[0].getAttribute('class').split(' ');
+    for(let i = 0; i < active.length; i++) {
+        if(active[i] == 'active') {
+            menu[0].classList.remove("active");
+        } else {
+            menu[0].className += ' active';
+        }
+    }
+}
+
+// back to top
+window.onscroll = function() {scrollFunction()};
+
+const scrollFunction = () => {
+    if(window.scrollY > 200) {
+        document.querySelectorAll('.backTop')[0].style.display = 'flex';
+    } else {
+        document.querySelectorAll('.backTop')[0].style.display = 'none';
+    }
+}
+
+const animateToTop = () => {
+    let scrollToTop = window.setInterval(function() {
+        let pos = window.pageYOffset;
+
+        if(pos > 0 && pageYOffset >= 10) {
+            window.scrollTo(0, pos - 20);
+        } else {
+            window.clearInterval(scrollToTop);
+        }
+    }, 9);
+}
+
+//slide
+const slideDelay = 3000;
+let curSlide = 0;
+
+const showSlide = (parent, slideIndex, autoplay) => {
+    if(typeof slideIndex == 'undefined' || slideIndex == null) {
+        slideIndex = 0;
+    }
+    let sliders = document.querySelectorAll('.'+parent)[0].children[0];

```

Comment:
```suggestion
    let sliders = document.querySelector('.' + parent).children[0];

```
e để ý cách viết code cho sạch và đẹp nhé